gym_minigrid/envs/fourrooms.py

```diff
@@ -66,7 +66,7 @@ def _gen_grid(self, width, height):
         else:
             self.place_obj(Goal())
 
-        self.mission = 'Reach the goal'
+        self.mission = 'reach the goal'
```

Comment:
Is there a particular reason for this change?

Reply:
All level missions use lower-case letters, except this one. This change was for consistency. It also helps with `DictObservationSpaceWrapper`.

Reply:
Could you add this to the PR changes as this could be important for some users

Reply:
I'm not sure I understand what you mean by "add this to the PR"

Reply:
Sorry I didn't explain it well, could you add to the top of the pull request where you have listed all of the major changes this as well